Get note source

sagotch opened the pull request:
Fix for #776

hgouraud committed:
Factorise wiki.wi get_note_source

Invert false/true in get_note_source
Removed the <p>…</p> strip
Add missing %xxx_source;

a2line committed:
Non approximative birth/death allowing baptism/burial in tables

Add fsources/fnotes on nb_children

a2line committed:
Fix spouse.index for implexes

a2line committed:
Add dl/dd/dt in whitelist for Bootstrap HTLM sanitizer in popover

hgouraud committed:
Remove boldface in popup titles. Missing > in ancsosa body_prop

a2line committed:
Hide notes, sources and corner implex with cgl

a2line committed:
Add gender and fix generation row on cgl

Commit:
perso.ml: Fixed wrong conf.no_note for sources display.

a2line commented:
When you don't need to modify the first commits of the PR and only want to add some modifications after them, don’t hesitate to push your modification on the PR itself. You don't need to close and push a new branch each times for that. I pushed on lledieu's PR a few commits a month ago. You don't loose the sha of the commits, the title, the comments and the labels on the PR and don't create duplicate PRs. When you wan't to modify the whole PR you can even push --force to that branch but be sure to have the original branch on your side as a precaution!

Please try to reopen #776 then
```
git checkout pr/origin/776
git cherry pick 0ff3cf5fe372088a8b97f1b8524ea288e77eab0e
git push hgouraud HEAD:get-note-source
```
Same thing, nope? I'm quite sure it works and if so, close this #839. I think this is a good practice!

sagotch commented:
I removed a commit. I do not think it is a good practice to push other's branches if it is not expected by its author, especially if you need to push force. The best is to submit PR to these other branches. I could if I did not rebased and removed a commit, but I can't.

a2line commented:
Ok, sorry, I did not saw that you removed a commit at 3 a.m. If you add only a commit it's not harmfull at all, moreover if it's a very little correction that is linked and you wan't to merge behind the PR (it saves some operation for you). I'm ok to merge this one. I just write here that the removed commit is the one on “capitalized capitale” because I can't remember where was this side effect that made us do it at the moment, and it's probably still here. @hgouraud can perhaps tell us.

a2line commented:
I did a typo on my commit name “HTLM” instead of HTML. For your last one, lots of writing guides for the name of the commits says to use a capital at beginning and no ending point, I'll write `Fixed wrong conf.no_note for sources display`. I don't think it's useful to keep the modified file in the commit name, but I also understand it can help to see things clearer/quicker. Tell me if you want me to reword those, I can do it.

hgouraud reviewed (dismissed):
ok for me (the fail with Travis and 4.08 can probably be ignored!!)

hgouraud commented:
back to #776
